include/hermite_multidimensional.hpp

```diff
@@ -21,6 +21,9 @@
 #pragma once
 #include <stdafx.h>
 #include <assert.h>
+#include <algorithm>
+#include <cstring>
```

Comment:
(how come these are added?)

Reply:
Without them the code failed to compile. Figuring out that this was needed was what took the most time. Maybe the reason why they were not needed before was that having eigen in the linking step automatically provided with things like `memcpy` which are needed here. But to be honest I don't know. If you remove these two the compilation will fail.

Reply:
Nice 🕵️ work, I definitely would have not known to add these